tests/events/Makefile.BSD

```diff
@@ -9,6 +9,7 @@ LIBCPERCIVA_DIR	=	../..
 
 # Main test code
 SRCS	=	main.c
+SRCS	+=	events_counter.c
```

Comment:
I'm working on an "event counter network server" to test `event_network.c`, so it makes sense to share the "event counter" functionality between the two binaries.